server.js

```diff
@@ -21,13 +14,135 @@ const app = express()
 app.use(cors())
 app.use(express.json())
 
-// Start defining your routes here
+// Start defining routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send({
+    Welcome: "Welcome to my Netflix API",
+    documentation: "https://documenter.getpostman.com/view/18068162/UVR4PW9m"
+  })
+})
+
+// route provides all endpoints
+app.get('/endpoints', (req, res) => {
+  res.json({
+    response: listEndpoints(app),
+    success: true
+  })
+})
+
+// route provides a sorted list of all countries that are in netflixData
+app.get('/countries', (req, res) => {
+  const countries = Array.from(new Set(netflixData.map(item => item.country))).sort()
+
+  res.json({
+    response: countries,
+    success: true
+  })
+})
+
+// route with all shows (both movies and other) from the provided country
+app.get('/countries/:country', (req, res) => {
+  // req is an object with properties such as query and params, 
+  // both query and params are empty objects from the beginning 
+  // every query parameter consists of key and value
+  const { country } = req.params
+  const contentByCountry = netflixData.filter((item => item.country.toLowerCase() === country))
+
+  if (contentByCountry.length === 0) {
+    res.status(404).json({
+      response: 'Sorry, but there is no content for this country',
+      success: false
+    })
+  } else {
+    res.json({
+      response: contentByCountry,
+      success: true
+    })
+  } 
+})
+
+// route provides all movies and has the possibility to query for director, year and actor. You can also do pagination by setting page & limit as query parameters
+app.get('/movies', (req, res) => {
+  const { director, year, actor, page, limit } = req.query
+
+  const movies = netflixData.filter(item => item.type === "Movie")
```

Comment:
You only have movies, not TV shows in your project so that is why you are filtering by type :)